week-4/F-object-keys/exercise-part-1.js

```diff
@@ -15,11 +15,11 @@ var mentorsAges = {
 
 // ONLY EDIT BELOW THIS LINE
 
-var mentorsNames = ;
+var mentorsNames = Object.keys(mentorsAges);
 
-var mentorsNamedCapitalised = ;
+var mentorsNamedCapitalised = mentorsNames.map(function(arr){return arr.toUpperCase()});
```

Comment:
This is correct. Can you think of a more meaningful name for your callback argument? `arr` suggests it's an array you expect here.